VideoLocker/src/main/java/org/edx/mobile/util/images/ErrorUtils.java

```diff
@@ -35,4 +40,34 @@ public static String getErrorMessage(@NonNull Throwable ex, @NonNull Context con
         }
         return errorMessage;
     }
+
+    /**
+     * Deliver the appropriate error message to their callbacks.
+     *
+     * @param ex The exception that occurred
+     * @param context A Context
+     * @param taskMessageCallback A TaskMessageCallback to deliver the error to, or null if not
+     *                            available.
+     * @param bannerDisplayCallback A BannerDisplayCallback to display the error message as a
+     *                              banner if appropriate, or null if not available.
+     */
+    public static void displayErrorMessage(@NonNull Throwable ex, @NonNull Context context,
+                                           @Nullable TaskMessageCallback taskMessageCallback,
+                                           @Nullable BannerDisplayCallback bannerDisplayCallback) {
+        if (bannerDisplayCallback != null) {
+            if (ex instanceof HttpConnectivityException) {
+                bannerDisplayCallback.showBanner(BannerType.OFFLINE);
+                return;
+            }
+            if (ex instanceof HttpResponseStatusException &&
+                    ((HttpResponseStatusException) ex).getStatusCode() == 426) {
```

Comment:
Yeah, I was thinking of creating a class for all the HTTP error constants, but didn't get the time for it. I'll try to do this at the end.